src/plugins/intel_gpu/src/runtime/layout.cpp

```diff
@@ -623,4 +617,32 @@ ov::PartialShape layout::transform(cldnn::format new_fmt) const {
     return ov::PartialShape(new_shape);
 }
 
+// Check a reorder is 1d along feature axis. Or feature size fits to inner block size of feature axis
+static inline bool check_redundant_1d_along_feature(layout const& l1, layout const& l2) {
+    if (!l1.data_padding && !l1.data_padding && l2.data_type == l1.data_type && l2.count() == l1.count()) {
+        // Get inner block size
+        if (static_cast<size_t>(l2.feature()) == l1.count() && l2.feature() == l1.feature() &&
+            static_cast<size_t>(l1.feature()) == l2.count() &&
+            (format::traits(l1.format).block_sizes.size() == 1 || format::traits(l2.format).block_sizes.size() == 1)) {
+            return true;
+        }
```

Comment:
If the tensor size is [1,4,1,1] and input format is fsv4 and output format is fsv16, it is not compatible. But this code will return true...

Reply:
Discussion : need to consider input-output in are_layout_identical

Reply:
Solution : instead of checking input-output, check a feature size is multiple of bigger inner block.
i.e. fsv4 and fsv16 => multiple of 16 / fsv16 and fsv32 => multiple of 32

Reply:
Hi @byungilm，  Recently I find this patch cannot remove the reoder of this pattern:
input float16 [1,144,1,1] b_fs_yx_fsv32 --> reorder -->output float16 [1,144,1,1]  b_fs_yx_fsv16,  even the data layout is 1d along feature.  144 is not the multiple of inner block fsv32.  I try to remove this logical judgment “(l2.feature() % max_inner_blk == 0)”, but this reoder still cannot be removed.  Can we further fuse this reoder into the previous node ?